src/createActionListenerMiddleware.ts

```diff
@@ -202,55 +186,48 @@ export function createActionListenerMiddleware<
 
     const listeners = listenerMap[action.type]
```

Comment:
Would it maybe make sense to split it into _two_ maps, one for "before" and one for "after", so that we only iterate through listeners relevant to that phase?

Reply:
We could do that, but I wouldn't worry about it too much. It probably doesn't matter unless someone has hundreds of listeners for the same action type - and even then it's only a factor of 2.

Reply:
Fair point.  I trust that Node can iterate through a bunch of things fairly quickly :)